lua/astrocommunity/pack/hyprland/init.lua

```diff
@@ -0,0 +1,65 @@
+---@type LazySpec
+return {
+  -- mason ensure install hyprls
+  {
+    "WhoIsSethDaniel/mason-tool-installer.nvim",
+    optional = true,
+    opts = function(_, opts)
+      opts.ensure_installed = require("astrocore").list_insert_unique(opts.ensure_installed, { "hyprls" })
+    end,
+  },
+
+  {
+    "AstroNvim/astrocore",
+    optional = true,
+    ---@type AstroCoreOpts
+    opts = {
+      -- add "hyprlang" file tpye
+      filetypes = {
+        extension = {
+          hl = "hyprlang",
+        },
+        filename = {
+          ["hyprland.conf"] = "hyprlang",
+        },
+        pattern = {
+          ["*/hypr/*.conf"] = "hyprlang",
+          ["hypr*.conf"] = "hyprlang",
```

Comment:
```suggestion
          [".*/hypr/.*.conf"] = "hyprlang",
          ["hypr.*.conf"] = "hyprlang",
```

These patterns do not work, they are real regular expressions so this fixes them.

Reply:
yeah they have